pkg/app/cassandra.go

```diff
@@ -55,7 +56,6 @@ func NewCassandraInstance(name string) App {
 			RepoURL:  helm.IncubatorRepoURL,
```

Comment:
Do we get any deprecation warning when we install cassandra from incubator help repo.

Reply:
No but I have changed the repo and url to bitnami now.